design/manual-authn-and-authz/AUTHORIZATION.md

```diff
@@ -0,0 +1,52 @@
+# Manual Authorization
+
+_Note: This guide expects that Authentication was set up as describe in the [AUTHENTICATION.md](AUTHENTICATION.md) guide_
+
+**This document describes a temporary workaround for handling Authorization manually.** 
+**The long term plan for Strimzi is to have custom resources to define ACL rules for Authorization.**
+**Until this is available, this temporary workaround will be available.**
+**There is no long term commitment to support this workaround.**
+**After the final implementation of the operator for Authorization is implemented, this workaround will be disabled and and it will not be possible to use it any more.**
```

Comment:
After the final implementation of the operator for Authorization this workaround will be disabled and it will not be possible to use it any more.

i.e. fix the two "and"s and "implementation of the Operator is implemented" is implementing a bit too much